addons/website_sale_digital/static/src/js/sidebar.js

```diff
@@ -0,0 +1,14 @@
+odoo.define('web.sidebar.digital', function (require) {
+"use strict";
+
+var Sidebar = require('web.Sidebar');
+
+Sidebar.include({
+    get_attachement_domain: function() {
```

Comment:
Removes the downloadable content from the product in the "sidebar" (which is at the top now)
(the "X attachment(s)" dropdown)